### PR TITLE
allow specifying repo path.

### PR DESCRIPTION
**Pull request by k4leung4**
git repo may not be at base github_workspace.

Signed-off-by: Kenny Leung <kleung@chainguard.dev>

### CODE REVIEW

File: nodiff/README.md [hunk 1]
```diff
@@ -28,5 +28,6 @@ steps:
 # Flag any differences from gofmt.
 - uses: chainguard-dev/actions/nodiff@main
   with:
+    path: "src/github.com/${{ github.repository }}
```
Comment (mattmoor) [inline nodiff/README.md:31]:
```suggestion
    path: "src/github.com/${{ github.repository }}"
```

File: eof-newline/action.yaml [hunk 2]
```diff
@@ -5,7 +5,12 @@ name: 'EOF Newline'
 description: |
   This action flags files that are missing newlines at EOF.
 
-inputs: {}
+inputs:
+  path:
```
Comment (mattmoor) [inline eof-newline/action.yaml:9]: add to this README?

File: trailing-space/action.yaml [hunk 3]
```diff
@@ -5,7 +5,12 @@ name: 'Trailing Whitespace'
 description: |
   This action flags occurrences of trailing whitespace at line endings.
 
-inputs: {}
+inputs:
+  path:
+    description: |
+      The path where the git repository resides.
+    required: true
+    default: ""
```
Comment (mattmoor) [inline trailing-space/action.yaml:13]: same

File: eof-newline/action.yaml (hunk 2 above)
Comment (k4leung4) [reply eof-newline/action.yaml:9]: done

File: trailing-space/action.yaml (hunk 3 above)
Comment (k4leung4) [reply trailing-space/action.yaml:13]: done

**Review by mattmoor — APPROVED** [review]

**Pull request merged**